tox.ini

```diff
@@ -39,6 +39,7 @@ deps =
     isodate
     setuptools
     boto3
+    pytest
```

Comment:
i don't think this is needed. it isn't running unit tests as part of pre-commit.

Reply:
Discussed offline: It's required by test_graphkv.py so that we can run parameterized tests. The linter checks the tests/ directory, hence I added it as a dependency for the pre-commit hook.